client/src/app/components/AboutUs/about-us.css

```diff
@@ -0,0 +1,80 @@
+.about-us {
+  text-align: center;
+}
+
+.aboutBackground {
+  position: absolute;
+  left: 0;
+  z-index: 1;
+  background-image: url("https://images.pexels.com/photos/1068523/pexels-photo-1068523.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260");
```

Comment:
Images should ideally be served by our server, not from the internet. We can leave it like this for now.